tests/analytics.bats

```diff
@@ -51,7 +56,7 @@ setup() {
   [ "$status" -eq 0 ]
 }
 
-@test "[analytics] Set no auto-create on Solr" {
+@test "[analytics] Set auto-create fields on Solr to true" {
```

Comment:
\* to false (see https://github.com/ebi-gene-expression-group/index-gxa/blob/develop/bin/gxa-index-set-no-autocreate.sh)

By the way, this is analogous to the test `@test "[bioentities] Disable automatic field generation in the Solr collection"` in L164, so I’d use the same wording to avoid confusion.

Also, you might want to read this: https://solr.apache.org/guide/7_1/schemaless-mode.html#disabling-automatic-field-guessing. The problem here is that automatic field guessing conflicts with [signature (i.e. unique key) generation in the `id` field](https://solr.apache.org/guide/7_1/de-duplication.html), so you can’t have both enabled unless you want to provide your own `id` value.